cardano-cli/test/cardano-cli-golden/files/golden/help.cli

```diff
@@ -7963,6 +7964,19 @@ Usage: cardano-cli conway query proposals [--cardano-mode [--epoch-slots SLOTS]]
   submitted during the current epoch are excluded, as they cannot be ratified
   until the next epoch.
 
+Usage: cardano-cli conway query era-history 
+                                              [--cardano-mode
+                                                [--epoch-slots SLOTS]]
+                                              ( --mainnet
+                                              | --testnet-magic NATURAL
+                                              )
+                                              --socket-path SOCKET_PATH
+                                              [--volatile-tip | --immutable-tip]
+                                              --out-file FILEPATH
+
+  Obtains the era history data. Era history data can be used to calculate slot
```

Comment:
This needs a little more fleshing out. Why is calculating slot times useful? Where will this be used?

Reply:
Addressed here: https://github.com/IntersectMBO/cardano-cli/pull/1079/commits/6b0b9edc1aae8c707350c8818a081ac2ee986a2e